config-paths.js

```diff
@@ -0,0 +1,23 @@
+const { app } = require('electron');
+const { homedir } = require('os');
+const { resolve } = require('path');
+const { existsSync } = require('fs');
+
+const home = homedir();
+const root = resolve(app.getPath('appData'), 'HyperTerm');
```

Comment:
Nitpick, but `appData` => `userData` and you could simplify it to: `const root = app.getPath('userData');`